plover/gui_qt/dictionaries_widget.py

```diff
@@ -377,6 +378,38 @@ def _create_new_dictionary(self):
         self._update_dictionaries(dictionaries, keep_selection=False,
                                   loaded_dictionaries=self._loaded_dictionaries)
 
+    def on_table_context_menu(self, row, global_pos):
+        if row == -1:
+            # when the user right-clicks in the empty area of the table
+            return
+
+        menu = QMenu(self)
+        saveAsAction = QAction("Save a Copy As...", self)
```

Comment:
- Let's switch to single quotes like you suggested
- The string inside this call should be translated with `_()`